include/MeshManager.hpp

```diff
@@ -19,49 +19,44 @@ namespace blitzdg {
       index_type NumVerts;
       index_type ElementType;
       index_type NumElements;
-      std::string CsvDelimeters;
-
+      std::string CsvDelimiters;
       index_type* ElementPartitionMap;
       index_type* VertexPartitionMap;
 
-      template<typename T>
-      void  readCsvFile(std::string csvFile, std::string delimiters, T* & result, index_type* & dims);
-
-      template<typename T>
-      void printArray(T* & arr, index_type numRows, index_type numCols);
-
     public:
       MeshManager();
 
-      index_type get_Index(index_type row, index_type col, index_type numCols);
+      static index_type get_Index(index_type row, index_type col, index_type numCols);
```

Comment:
nice catch here...

Reply:
Thanks. It was fine as a non-static member, but by making it static I was able to move the private members printArray and readCsvFile out of the MeshManager class. Since they weren't using any internals of the class, I felt like they didn't need to be in there.